src/com/company/Main.java

```diff
@@ -9,19 +10,20 @@ public static void main(String[] args) {
 
         Scanner sc = new Scanner(System.in);
 
-        System.out.println("Do you want to know actual date and time ?" + " Enter Yes/No");
+        System.out.println("Do you want to know actual date and time ? Enter Yes/No");
         String ans = sc.nextLine();
 
+        //inquiry
         if (ans.equals("Yes")) {
 
-            Calendar res = Calendar.getInstance();
-
-            System.out.println("Current Date and Time :");
-            System.out.format("%tB %te, %tY%n", res, res, res);
-            System.out.format("%tl:%tM %tp%n", res, res, res);
+            LocalDate date = LocalDate.now();         // output Date/Time in Java 8
```

Comment:
Good. Please, look at DateTimeFormatter too. It helps to use patterns for output and many more toher things. It's useful to know different ways and approaches.

Reply:
Done